pages/tasks/index.js

```diff
@@ -121,7 +125,7 @@ const TaskPage = ({ issues }) => {
                     </div>
                     <div className="flex flex-col gap-4">
                       {filteredIssues.map((issue) => (
-                        <IssueCard key={issue.github_id} issue={issue} t={t} userInfo={userAuth} />
+                        <IssueCard key={issue.github_id} issue={issue} userInfo={userAuth} />
```

Comment:
pass userAuth as parameter is very weird, avoid doing that since we can get this information globally